src/Aspire.Hosting/Seq/SeqBuilderExtensions.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Publishing;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Seq server resources to the application model.
+/// </summary>
+public static class SeqBuilderExtensions
+{
+    // The path within the container in which Seq stores its data
+    const string SeqContainerDataDirectory = "/data";
+
+    /// <summary>
+    /// Adds a Seq server resource to the application model. A container is used for local development.
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name to give the resource.</param>
+    /// <param name="addToManifest">Should this resource be added to the manifest.</param>
+    /// <param name="port">The host port for the Seq server.</param>
+    /// <param name="seqDataDirectory">Host directory to bind to Seq's data directory. This must already exist.</param>
+    public static IResourceBuilder<SeqResource> AddSeq(
+        this IDistributedApplicationBuilder builder,
+        string name = "Seq",
+        bool addToManifest = false,
```

Comment:
this isn't a pattern we have anywhere. I'd remove it.

Reply:
It's hard for me to see which line you are commenting on. Do you mean `addToManifest`? 

The goal is to default to Seq being a development time resource, because most organizations have a single Seq server. You generally wouldn't deploy it with Aspire, but you might want to.

Reply:
Right, I mean `addToManifest`. I would keep the model consistent and remove this parameter. There are ways to model "container in dev and external connection string why deployed" so I would leave this as is.

Reply:
OK. Done.